src/ChannelServer/Util/GmCommands.cs

```diff
@@ -49,7 +49,7 @@ public GmCommands()
 			Add("speed", "<speed>", HandleSpeed);
 			Add("iteminfo", "<name>", HandleItemInfo);
 			Add("monsterinfo", "<name>", HandleMonsterInfo);
-			Add("go", "<destination>", HandleGo);
+			Add("tele", "<destination>", HandleTele);
```

Comment:
I think it'd be preferred that this was renamed to warp, just to be consistent with other aura projects ;)